### PR TITLE
Migrate content from std_msgs

### PR DESCRIPTION
**Pull request by tfoote**
This if following up https://github.com/ros2/common_interfaces/issues/89

### CODE REVIEW

File: msg/Bool.msg [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# This is an example message of using a primative datatype, bool.
+# If you want to test with this that's fine, but if you are deploying
+# it into a system you should create a semantically meaningful custom message.
```
Comment (hidmic) [inline msg/Bool.msg]: @tfoote consider
```suggestion
# it into a system you should create a semantically meaningful message type.
```
same elsewhere.

File: msg/Bool.msg [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+# If you want to embed it in another message use the primative not this message.
```
Comment (hidmic) [inline msg/Bool.msg]: @tfoote consider:
```suggestion
# If you want to embed it in another message, use the primitive data type instead.
```
same elsewhere.

File: msg/Bool.msg [hunk 3]
```diff
@@ -0,0 +1,5 @@
+# This is an example message of using a primative datatype, bool.
```
Comment (hidmic) [inline msg/Bool.msg]: @tfoote consider
```suggestion
# This is an example message of using a primitive datatype, bool.
```

**Comment by tfoote** [comment]
Good suggestions, updated.

**Review by jacobperron — APPROVED** [review]

File: msg/MultiArrayLayout.msg [hunk 4]
```diff
@@ -0,0 +1,30 @@
+# This is an example of using complex datatypes.
+# It is not recommended to use directly.
+# To use a similar datastruct please define a custom message with appropriate semantic meaning.
```
Comment (hidmic) [inline msg/MultiArrayLayout.msg:3]: @tfoote `datastruct` -- In other places we talk about `data types` in a similar, almost equivalent way. Consider standardizing the terminology.

**Review by hidmic — APPROVED** [review]

File: msg/MultiArrayLayout.msg (hunk 4 above)
Comment (tfoote) [reply msg/MultiArrayLayout.msg:3]: I specifically chose datastruct here to refer to the structure as opposed to the type as the guidance is to make a new type (indirectly by the new message).

**Comment by ros-discourse** [comment]
This pull request has been mentioned on **ROS Discourse**. There might be relevant details there:

https://discourse.ros.org/t/soss-a-whole-new-approach-to-your-ros-1-ros-2-bridge/17040/18

**Pull request merged**